homeassistant/components/mystrom/light.py

```diff
@@ -78,48 +78,14 @@ class MyStromLight(LightEntity):
     def __init__(self, bulb, name, mac):
         """Initialize the light."""
         self._bulb = bulb
-        self._name = name
-        self._state = None
-        self._available = False
-        self._brightness = 0
         self._color_h = 0
         self._color_s = 0
-        self._mac = mac
-
-    @property
-    def name(self):
-        """Return the display name of this light."""
-        return self._name
-
-    @property
-    def unique_id(self):
-        """Return a unique ID."""
-        return self._mac
-
-    @property
-    def brightness(self):
-        """Return the brightness of the light."""
-        return self._brightness
-
-    @property
-    def hs_color(self):
-        """Return the color of the light."""
-        return self._color_h, self._color_s
-
-    @property
-    def available(self) -> bool:
-        """Return True if entity is available."""
-        return self._available
-
-    @property
-    def effect_list(self):
-        """Return the list of supported effects."""
-        return MYSTROM_EFFECT_LIST
-
-    @property
-    def is_on(self):
-        """Return true if light is on."""
-        return self._state
+        self._attr_name = name
+        self._attr_available = True
```

Comment:
Previous default was `False`
```suggestion
        self._attr_available = False
```